src/Functions/hilbertEncode.cpp

```diff
@@ -0,0 +1,59 @@
+#include <Functions/hilbertEncode.h>
```

Comment:
The same as for `hilbertDecode`, it's definitely better to put everything into the single `.cpp` file, as `.cpp` file has not much sense now